### PR TITLE
Flutter path fix

### PR DESCRIPTION
**Pull request by Kanan-Bitrise**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [x] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [x] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

File: roles/flutter/tasks/main.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   unarchive:
     src: https://storage.googleapis.com/flutter_infra/releases/stable/macos/flutter_macos_{{ flutter_version }}-stable.zip
     dest: "{{ flutter_home }}"
-    mode: 0770
+    mode: '0770'
```
Comment (fehersanyi-bitrise) [inline roles/flutter/tasks/main.yml:6]: use double quotes please

Comment (Kanan-Bitrise) [reply roles/flutter/tasks/main.yml:6]: the documentation says use single quotes

Comment (Kanan-Bitrise) [reply roles/flutter/tasks/main.yml:6]: https://docs.ansible.com/ansible/latest/modules/file_module.html

File: roles/flutter/templates/patch.j2 [hunk 2]
```diff
@@ -1 +1,3 @@
-export PATH=$PATH:{{ flutter_home }}/flutter/bin
+export PATH="$PATH:{{ flutter_home }}/flutter/bin"
```
Comment (lmesz-bitrise) [inline roles/flutter/templates/patch.j2:1]: I would add flutter to the flutter home

**Pull request merged**